src/eko/kernels/QEDnon_singlet.py

```diff
@@ -0,0 +1,364 @@
+# -*- coding: utf-8 -*-
+"""Collection of QED non-singlet EKOs."""
+import numba as nb
+import numpy as np
+
+from .. import beta
+from . import evolution_integrals as ei
+from . import non_singlet, utils
+
+
+@nb.njit(cache=True)
+def lo_aem1_exact(gamma_ns, a1, a0, aem, nf):  # lo refers to the order in as1
```

Comment:
let's enforce our new naming scheme also here: `as1_aem1_exact` (and similar of course)

Reply:
(of course this also cancels the comment)

Reply:
actually the new scheme would be `as1aem1_exact` (without the underscore)